### ISSUE TITLE
Improve transparency that you aren't official

### ISSUE THREAD

**Issue by KobeW50** [revancedapps/revanced.net#3]
Hi there. I'm an active helper in the ReVanced and ReVanced Extended communities.

You state in the [#README](https://github.com/revancedapps/revanced.net#readme) of this repo the following 2 quotes:

"At revanced.net, our primary goal is to offer a trustworthy platform where everyone can conveniently download various APKs related to ReVanced projects, as well as other modded APKs. Our aim is to provide clean and efficient installation links for your convenience."

"revanced.net will gradually release source codes related to our automated build processes through continuous delivery. This strategic step is taken to eliminate any unnecessary misunderstandings regarding our website and its operations. Our ultimate aim is to build trust and confidence among our users."

Unfortunately, much must be done if you would like to achieve this goal of trust and transparency. The site and repository poorly relay the fact that the revanced.net project is unofficial. A user would need to scroll to the bottom of the FAQ on the site to see this, which isn't an ideal design if trust is what you are looking to achieve. A prominent and bold acknowledgement that you aren't official on the home page will go a long way in providing comfort and assurance to users that you are trustworthy.

Furthermore, the ReVanced.net Manager and all patched apks on the site are branded as if they are the official apps. I suggest renaming the text on the download buttons to emphasize that these are apks made by the community (which implies that they aren't official). For example, you can rename the download button for the ReVanced.net Manager to "Community ReVanced Manager".

Thank you for reading

**Comment by KobeW50** [comment]
> Thank you for your empathy and feedback on modifying the content. We will use these suggestions to make some adjustments that will increase the transparency of the website.

I see you have made many improvements to the site. Thank you 

One more suggestion:

It would make sense for you to add a link to this GitHub repository on the site so people can see that it is open-source, read the code, and help contribute

**Comment by bersbersbers** [comment]
"official repository _for the homepage source code_ of revanced _.net_" still sounds rather sketchy, as if someone wanted users to misread this as "official repository of ... revanced" :)

**Comment by bersbersbers** [comment]
You are right about the space. (I had to add it to make the italic formatting via underscores work - I did not mean to change the meaning of the current text.)
